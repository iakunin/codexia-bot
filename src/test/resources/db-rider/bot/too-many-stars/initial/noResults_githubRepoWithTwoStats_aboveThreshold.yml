github_repo:
    -
        id: 1
        uuid: "groovy: UUID.randomUUID().toString()"
        external_id: "150713223"
        full_name: "test-project1/test-repo1"

codexia_project:
    -
        id: 5
        uuid: "groovy: UUID.randomUUID().toString()"
        external_id: 12
        coordinates: "test-project1/test-repo1"
        author: "some-first-author"
        deleted: null
        project_created_at: "2019-12-20 12:01:02"

github_repo_source:
    -
        id: 1
        uuid: "groovy: UUID.randomUUID().toString()"
        github_repo_id: 1
        source: "CODEXIA"
        external_id: "12"

github_repo_stat:
    -
        id: 2
        uuid: "groovy: UUID.randomUUID().toString()"
        github_repo_id: 1
        type: "GITHUB_API"
        stat: '{"type": "GITHUB_API", "stars": 10023}'
        created_at: "2020-02-22 14:15:16.112233"
    -
        id: 3
        uuid: "groovy: UUID.randomUUID().toString()"
        github_repo_id: 1
        type: "GITHUB_API"
        stat: '{"type": "GITHUB_API", "stars": 10111}'
        created_at: "2020-02-23 07:08:09.112233"

too_many_stars_result: []
