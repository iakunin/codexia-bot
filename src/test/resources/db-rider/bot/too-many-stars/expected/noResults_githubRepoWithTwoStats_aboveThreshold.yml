too_many_stars_result:
    -
        github_repo_id: 1
        github_repo_stat_id: 3

codexia_review:
    -
        codexia_project_id: 5
        author: "TOO_MANY_STARS"
        reason: "10111"
        text: >-
            The repo gained too many stars: 10111 (at 2020-02-23T07:08:09Z).
